webapp/channels/src/components/profile_popover/components/profile_popover_user_name.tsx

```diff
@@ -9,19 +9,22 @@ type Props = {
     username: string;
 }
 
-const UserName = ({
+const ProfilePopoverUserName = ({
     hasFullName,
     username,
 }: Props) => {
-    const userNameClass = classNames('overflow--ellipsis pb-1', {'user-profile-popover__heading': !hasFullName});
     return (
         <div
             id='userPopoverUsername'
-            className={userNameClass}
+            className={classNames({
```

Comment:
I think it'd be easier to read this if it were a ternary than using `classNames` since it's only going to use one class or the other

Reply:
yes changed